nemo/collections/asr/modules/conv_asr.py

```diff
@@ -382,35 +458,37 @@ def output_types(self):
         )
 
     def __init__(
-        self, feat_in, num_classes, emb_sizes=None, pool_mode='xvector', angular=False, init_mode="xavier_uniform",
+        self,
+        feat_in,
+        num_classes,
+        emb_sizes=[256],
```

Comment:
Do not use [256] in function. Use None, and check below if None then set emb_size = [256].

Reply:
using int